packages/api/pages/api/verses/[verseId]/lemma-resources.ts

```diff
@@ -0,0 +1,47 @@
+import { GetLemmaResourcesResponseBody } from '@translation/api-types';
+import createRoute from '../../../../shared/Route';
+import { Prisma, client } from '../../../../shared/db';
+
+export default createRoute<{ verseId: string }>()
+  .get<void, GetLemmaResourcesResponseBody>({
+    async handler(req, res) {
+      const verse = await client.verse.findUnique({
+        where: {
+          id: req.query.verseId,
+        },
+        include: {
+          words: {
+            orderBy: {
+              id: Prisma.SortOrder.asc,
+            },
+            include: {
+              form: true,
+            },
+          },
+        },
+      });
+
+      if (verse) {
+        const data = [];
+        for (const word of verse.words) {
+          const resources = await client.lemmaResource.findMany({
```

Comment:
This could query for duplicate lemmas if the same one is used more than once in a verse. Can we consolidate to make sure we are querying for unique lemmas?

Reply:
Done